models/DataObject/ClassDefinition/Data/ManyToManyObjectRelation.php

```diff
@@ -636,6 +637,10 @@ public function enrichLayoutDefinition(/* ?Concrete */ $object, /* array */ $con
 
                     $this->visibleFieldDefinitions[$field]['options'] = $fd->getOptions();
                 }
+
+                if ($fd instanceof DataObject\ClassDefinition\Data\BooleanSelect){
+                    $this->visibleFieldDefinitions[$field]['options'] = $fd->getOptions();
+                }
```

Comment:
Do we have to add those lines for other relations too? E.g. here https://github.com/pimcore/pimcore/blob/10.5/models/DataObject/ClassDefinition/Data/AdvancedManyToManyObjectRelation.php#L862-L871 
Possibly in other places as well?